test_track_writes randomly fails

###### Arch: amd64
###### Distributive: Debian 10
###### Filesystem: ext4, LVM
###### Device type: qcow2
###### Details:

Subtask of #166 

This is a very rare event, was seen only once on debian10 (amd64), LVM + qcow2 disk

```
======================================================================
FAIL: test_track_writes (test_snapshot.TestSnapshot)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/elastio/elastio-snap/tests/test_snapshot.py", line 74, in test_track_writes
    self.assertGreater(end_nr, start_nr)
AssertionError: 4 not greater than 4
```

--- a/.github/workflows/ci.yml
+++ b/.github/workflows/ci.yml
@@ -184,7 +184,7 @@ jobs:
         working-directory: ${{env.BOX_DIR}}
         # For now tests are taking up to 40 seconds per 1 fs on amd64 and about 2 minutes on arm64 or older 3.x kernels. But they can hang.
         # 20 minutes seems to be reasonable timeout.
-        timeout-minutes: 20
+        timeout-minutes: 25
 
       - name: Show debug information
         if: always()
@@ -201,7 +201,7 @@ jobs:
         working-directory: ${{env.BOX_DIR}}
         # For now tests are taking 10-20 seconds per 1 fs on amd64 and about 2 minutes on arm64. But they can hang.
         # 20 minutes seems to be reasonable timeout.
-        timeout-minutes: 20
+        timeout-minutes: 25
 
       - name: Show debug information
         if: always()
@@ -218,7 +218,7 @@ jobs:
         working-directory: ${{env.BOX_DIR}}
         # For now tests are taking up to 40 seconds per 1 fs on amd64 and about 2 minutes on arm64 or older 3.x kernels. But they can hang.
         # 20 minutes seems to be reasonable timeout.
-        timeout-minutes: 20
+        timeout-minutes: 25
 
       - name: Show debug information
         if: always()
@@ -257,7 +257,7 @@ jobs:
             vagrant ssh ${{env.INSTANCE_NAME}} -c "cd tests && sudo ./elio-test.sh -d /dev/vdb1 -f $fs"
           done
         working-directory: ${{env.BOX_DIR}}
-        timeout-minutes: 25
+        timeout-minutes: 30
 
       - name: Show debug information
         if: always()
@@ -272,7 +272,7 @@ jobs:
             vagrant ssh ${{env.INSTANCE_NAME}} -c "cd tests && sudo ./elio-test.sh -d /dev/vdb -d /dev/vdc -f $fs --lvm"
           done
         working-directory: ${{env.BOX_DIR}}
-        timeout-minutes: 25
+        timeout-minutes: 30
 
       - name: Show debug information
         if: always()
@@ -290,7 +290,7 @@ jobs:
             vagrant ssh ${{env.INSTANCE_NAME}} -c "cd tests && sudo ./elio-test.sh -d /dev/vdb -d /dev/vdc -f $fs --raid"
           done
         working-directory: ${{env.BOX_DIR}}
-        timeout-minutes: 25
+        timeout-minutes: 30
 
       - name: Show debug information
         if: always()

--- a/.github/workflows/ci.yml
+++ b/.github/workflows/ci.yml
@@ -184,7 +184,7 @@ jobs:
         working-directory: ${{env.BOX_DIR}}
         # For now tests are taking up to 40 seconds per 1 fs on amd64 and about 2 minutes on arm64 or older 3.x kernels. But they can hang.
         # 20 minutes seems to be reasonable timeout.
-        timeout-minutes: 25
+        timeout-minutes: 35
 
       - name: Show debug information
         if: always()
@@ -201,7 +201,7 @@ jobs:
         working-directory: ${{env.BOX_DIR}}
         # For now tests are taking 10-20 seconds per 1 fs on amd64 and about 2 minutes on arm64. But they can hang.
         # 20 minutes seems to be reasonable timeout.
-        timeout-minutes: 25
+        timeout-minutes: 35
 
       - name: Show debug information
         if: always()
@@ -218,7 +218,7 @@ jobs:
         working-directory: ${{env.BOX_DIR}}
         # For now tests are taking up to 40 seconds per 1 fs on amd64 and about 2 minutes on arm64 or older 3.x kernels. But they can hang.
         # 20 minutes seems to be reasonable timeout.
-        timeout-minutes: 25
+        timeout-minutes: 35
 
       - name: Show debug information
         if: always()
@@ -257,7 +257,7 @@ jobs:
             vagrant ssh ${{env.INSTANCE_NAME}} -c "cd tests && sudo ./elio-test.sh -d /dev/vdb1 -f $fs"
           done
         working-directory: ${{env.BOX_DIR}}
-        timeout-minutes: 30
+        timeout-minutes: 50
 
       - name: Show debug information
         if: always()
@@ -272,7 +272,7 @@ jobs:
             vagrant ssh ${{env.INSTANCE_NAME}} -c "cd tests && sudo ./elio-test.sh -d /dev/vdb -d /dev/vdc -f $fs --lvm"
           done
         working-directory: ${{env.BOX_DIR}}
-        timeout-minutes: 30
+        timeout-minutes: 50
 
       - name: Show debug information
         if: always()
@@ -290,7 +290,7 @@ jobs:
             vagrant ssh ${{env.INSTANCE_NAME}} -c "cd tests && sudo ./elio-test.sh -d /dev/vdb -d /dev/vdc -f $fs --raid"
           done
         working-directory: ${{env.BOX_DIR}}
-        timeout-minutes: 30
+        timeout-minutes: 50
 
       - name: Show debug information
         if: always()

--- a/tests/test_snapshot.py
+++ b/tests/test_snapshot.py
@@ -77,8 +77,7 @@ class TestSnapshot(DeviceTestCase):
         info = elastio_snap.info(self.minor)
         start_nr = info["nr_changed_blocks"]
 
-        with open(testfile, "w") as f:
-            f.write("The quick brown fox")
+        util.dd("/dev/urandom", testfile, 1, bs="512")
 
         self.addCleanup(os.remove, testfile)
         os.sync()

--- a/tests/test_snapshot.py
+++ b/tests/test_snapshot.py
@@ -77,7 +77,7 @@ class TestSnapshot(DeviceTestCase):
         info = elastio_snap.info(self.minor)
         start_nr = info["nr_changed_blocks"]
 
-        util.dd("/dev/urandom", testfile, 1, bs="512")
+        util.dd("/dev/urandom", testfile, 1, bs="1M")
 
         self.addCleanup(os.remove, testfile)
         os.sync()

